public/style/style.css

```diff
@@ -22,4 +22,53 @@ md-whiteframe .md-subheader {
   stroke: #000;
   stroke-width: 0.5px;
   opacity: 0.75;
-}
\ No newline at end of file
+}
+.slidecontainer {
```

Comment:
Does Travel not have its own dedicated css file? We should add one if it doesn't